### DISCUSSION TITLE
Lerna bootstrap is dead, now what?

### DISCUSSION THREAD

**Discussion by nir-cohe** [lerna/lerna#3904] [Q&A]
Hi,
I try to upgrade our pipline (Jenkins, mono-repo) with using Nx & Lerna, we had only lerna until now.
I try to figure out the steps, because, in the past we did lerna yarn exec & lerna bootstrap, that was install all the packages, and then making the linking between them.
today, according to doc, no more lerna bootstrap, so I tried to changed the command for:
```groovy
   stage('Install') {
                steps {
                    container(name: 'node') {
                        sh "yarn install"
                    }
                }
            }
``` 
and then:
```groovy
   stage('Build') {
                steps {
					container(name: 'node') {
						script {
                            buildEnvironment = getBuildEnvironment(currentBranch)
                            sh "yarn nx run-many -t build:${buildEnvironment}"
                        }
					}
				}
            }
``` 

And I can get an errors like:
Module not found: Error: Can't resolve 'translations' 

I guess the bootstrap step is missing, but lerna does not support it any more, any idea?

**Comment by JamesHenry** [comment]
Did you setup up `workspaces` in your package.json?

https://lerna.js.org/docs/legacy-package-management
